ini/stm32g0.ini

```diff
@@ -40,6 +40,22 @@ board_upload.offset_address = 0x08000000
 build_flags                 = ${stm32_variant.build_flags} ${stm32g0_I2C2.build_flags}
                               -flto
                               -Wl,--no-warn-rwx-segment
+
+[env:BTT_EBB42_V1_1_filament_extruder_CAN]
```

Comment:
This environment can be renamed to drop the “`filament_extruder`” name since that was only added to make it clear that this board did not work as a CAN board.